tests/test_run_llm2vec_service.py

```
@@ -56,3 +58,63 @@ def test_embedding_service_exposes_health_and_embed(tmp_path: Path) -> None:
     assert payload["normalised"] is True
     assert dummy.calls[0][0] == ["hello", "world"]
     assert dummy.calls[0][1] is True
+
+
+def test_embedding_service_ollama_backend(monkeypatch) -> None:
+    async def fake_encode_batch(self, texts, *, instruction=None):
```

Ruff (0.14.0)

Unused function argument: `self`

(ARG001)